<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSettingDialog</class>
 <widget class="QDialog" name="DeviceSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>462</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>455</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_back" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_set">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>272</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_audioIn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
border: 0px solid #bbb;
}

QSlider::sub-page:horizontal {
background: rgb(51,153,255); 
border-radius: 0px;
margin-top:13px;
margin-bottom:13px;
}

QSlider::add-page:horizontal {
background: rgb(204,204, 204);
border: 0px solid #777;
border-radius: 2px;
margin-top:13px;
margin-bottom:13px;
}

QSlider::handle:horizontal {
background: rgb(0,101,251); 
border: 1px solid rgb(0,101,251);  
width: 18px;
border-radius: 9px;
margin-top:6px;
margin-bottom:6px;
}

QSlider::handle:horizontal:hover {
background: rgb(10,101,251); 
border: 1px solid rgba(102,102,102,102);
width: 18px;
border-radius: 8px;
margin-top:6px;
margin-bottom:6px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>0</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>关闭</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:/image/close_2_normal.png);
border-radius:0px; 
	background-color: rgba(255, 255, 255, 0);
} 
QPushButton:hover{ 
image: url(:image/close_normal.png);
background-color:rgb(255,80,6); 
border-radius:12px; 
} 
QPushButton:pressed{ 
image: url(:image/close_hover.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>36</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,0,0,0">
           <property name="spacing">
            <number>12</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBox_audioIn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>36</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="widget" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>236</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>30</height>
                   </size>
                  </property>
                  <widget class="QProgressBar" name="progressBar_back">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>260</width>
                     <height>30</height>
                    </rect>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QProgressBar
{
       height:30;
background:transparent;
border:none;
	
}



QProgressBar::chunk
{
       background-color:#ffffff;
       width:5px;
	margin:0.5px 5px;
}</string>
                   </property>
                   <property name="value">
                    <number>100</number>
                   </property>
                   <property name="textVisible">
                    <bool>false</bool>
                   </property>
                  </widget>
                  <widget class="QProgressBar" name="progressBar">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>236</width>
                     <height>30</height>
                    </rect>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>236</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QProgressBar
{
       height:30;
background:transparent;
border:none;
	
}



QProgressBar::chunk
{
       background-color:#4b5cc4;
       width:5px;
	margin:0.5px 5px;
}</string>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                   <property name="textVisible">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_audioTest_stop">
                  <property name="minimumSize">
                   <size>
                    <width>98</width>
                    <height>36</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>98</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>停止测试</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_audioTest_start">
                  <property name="minimumSize">
                   <size>
                    <width>98</width>
                    <height>36</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>98</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>点击测试</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_note">
               <property name="text">
                <string>观察到音量有条有变化，则麦克风收音正常</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_virtual_audio_capture">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>录制声卡声音</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>36</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="topMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>视频画质:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_videoQuality">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>36</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>380</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="currentIndex">
                   <number>9</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>录屏保存路径：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lineEdit_videoPath">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>36</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_videoPath">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>36</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>98</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_confirm">
               <property name="minimumSize">
                <size>
                 <width>66</width>
                 <height>36</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_cancel">
               <property name="minimumSize">
                <size>
                 <width>66</width>
                 <height>36</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>DeviceSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>DeviceSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
